### ISSUE TITLE
Add loading animation or bigger loading bar when booting up WebTS for the first time

### ISSUE THREAD

**Issue by xtinah-w** [Microsoft/WebTemplateStudio#621]
Loading takes really long and some people experience confusion since there's little to no indication that it is actually launching. It takes a couple of seconds to boot up and we can lessen the confusion by adding the animation/larger loading bar.

**Comment by SahilTara** [comment]
Do you mean add it to the client and create an animation for it?

**Comment by xtinah-w** [comment]
Yeah, I mean to add a loading animation when developer launches WebTS

-> rewards dev for downloading WebTS and using it for the first time 
-> creates excitement upon launch

**Comment by SahilTara** [comment]
I have another suggestion for this can we also reallocate a lot of the install experience to this new UI? Right now our install experience is pretty bad. Its a 220+ Mb vsix. We can reduce the size to approx ~5 Mb and then download approx 70 Mb over the air for only that specific platform. This will let us also tackle concerns from #603. Also will reduce total size of our vsix from ~220 Mb to ~80 Mb. 

Any thoughts?
